src/puppetlabs/trapperkeeper/services/scheduler/scheduler_core.clj

```diff
@@ -1,12 +1,20 @@
 (ns puppetlabs.trapperkeeper.services.scheduler.scheduler-core
-  (:require [overtone.at-at :as at-at]
-            [clojure.tools.logging :as log]
-            [puppetlabs.i18n.core :as i18n]))
+  (:require [clojure.tools.logging :as log]
+            [puppetlabs.i18n.core :as i18n])
+  (:import (org.quartz.impl.matchers GroupMatcher)
+           (org.quartz.impl StdSchedulerFactory)
+           (org.quartz JobBuilder SimpleScheduleBuilder TriggerBuilder DateBuilder
+                       DateBuilder$IntervalUnit Scheduler JobKey SchedulerException)
+           (org.quartz.utils Key)))
 
-(defn create-pool
+(defn create-scheduler
   "Creates and returns a thread pool which can be used for scheduling jobs."
   []
-  (at-at/mk-pool))
+  ;; without the following property set, quartz does a version check automatically
+  (System/setProperty "org.quartz.scheduler.skipUpdateCheck" "true")
+  (let [scheduler (StdSchedulerFactory/getDefaultScheduler)]
+    (.start scheduler)
+    scheduler))
 
 (defn wrap-with-error-logging
```

Comment:
This is no longer used.

Reply:
I didn't know if anything outside the library might be using it since it is a utility thing.  I'll take it out and ask forgiveness later. :)